docs/source/reference.rst

```diff
@@ -3,14 +3,4 @@
 API reference
```

Comment:
What is actually the purpose of the reference.rst? I thought that this is a dump of everything of the library which I find useful.

Reply:
It's a dump of _everything_, kind of like an index for the whole package. I'm leaving it empty, for now, and we can expand later.